.cspell.json

```diff
@@ -1,241 +0,0 @@
-{
```

Comment:
This seems to be a generic file without special relation or customization to this project :flamingo: